### PR TITLE
common: scripts/README.* revamp for readability

### PR DESCRIPTION
**Pull request by janekmi**
No updates or fixes. Just readability.

---
This change is [Reviewable](https://reviewable.io/reviews/pmemhackathon/2021-04-21.web/5)

### CODE REVIEW

**Comment by ldorau** [comment]
Looks good to me :lgtm: :+1:

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+after that, the flow of the admin steps is:
+
+## foreach hackathon:
+
+1. create a local clone of the hackathon repo in `/home/hackathon`, e.g.:
+
+```sh
+cd /home/hackathon
+git clone https://githib.com/pmemhackathon/2019-07-07
```
Comment (ldorau) [inline scripts/README.md]: github

Comment (janekmi) [reply scripts/README.md]: Done.

Comment (ldorau) [reply scripts/README.md]: OK

**Comment by ldorau** [comment]
LGTM :+1:

**Review by ldorau — COMMENTED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmemhackathon/2021-04-21.web/5)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @janekmi)

**Review by lukaszstolarczuk — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmemhackathon/2021-04-21.web/5)* status: all files reviewed, 2 unresolved discussions (waiting on @janekmi)

**Review by lukaszstolarczuk — COMMENTED** [review]
I believe these 2 issues I've left can be updated in the next PR - #6

> *[Reviewable](https://reviewable.io/reviews/pmemhackathon/2021-04-21.web/5#-:-Mkr8wUCArB90JeuzB-9:b-cppdo8)* status: all files reviewed, 2 unresolved discussions (waiting on @janekmi)

**Review by grom72 — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pmemhackathon/2021-04-21.web/5#-:-NBGh9kG5aQb_hjOfm5u:bnfp4nl)* status: all files reviewed, 2 unresolved discussions (waiting on @janekmi)

**Review by grom72 — COMMENTED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmemhackathon/2021-04-21.web/5)* status: all files reviewed, 2 unresolved discussions (waiting on @janekmi)

**Comment by ldorau** [comment]
@igchor Could you move this PR to the new repo or close it?

**Comment by lukaszstolarczuk** [comment]
@ldorau I created new PR, as you suggested, pls review there so we could merge it there :)

**Pull request closed without merging**